### PR TITLE
Minor fixes.

### PR DESCRIPTION
**Pull request by yihuang**
## Type of change

- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🛠 New feature (non-breaking change which adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🏭 Refactoring that does not change existing functionality but does improve things like code readability, structure etc
- [ ] 🔨 New or improved tests for existing code
- [ ] ⛑ git-flow chore (backport, hotfix, etc)

## Developer checklist

- [ ] I have read the [style guide](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/style-guide.md) document, and my code follows the code style of this project.
- [ ] If my code deals with exceptions, it follows the [guidelines](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/exceptions.md).
- [ ] I have updated any documentation accordingly, if needed. Documentation changes can be reflected in opening a PR on [cardanodocs.com](https://github.com/input-output-hk/cardanodocs.com), amending the inline [Haddock](https://www.haskell.org/haddock/) comments, any relevant README file or one of the document listed in the [docs](https://github.com/input-output-hk/cardano-sl/tree/develop/docs) directory.

## Testing checklist

- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: lib/src/Pos/Network/Block/Retrieval.hs [hunk 1]
```diff
@@ -114,7 +114,7 @@ retrievalWorker pm txpConfig diffusion = do
         logDebug $ "handleContinues: " <> pretty hHash
         classifyNewHeader pm header >>= \case
             CHContinues ->
-                void $ getProcessBlocks pm txpConfig diffusion nodeId (headerHash header) [hHash]
+                void $ getProcessBlocks pm txpConfig diffusion nodeId hHash [hHash]
```
Comment (erikd) [inline lib/src/Pos/Network/Block/Retrieval.hs]: What problem is this fixing? To me this actually looks incorrect. The function:
```
getProcessBlocks
    :: forall ctx m.
       ( BlockWorkMode ctx m
       , HasMisbehaviorMetrics ctx
       )
    => ProtocolMagic
    -> TxpConfiguration
    -> Diffusion m
    -> NodeId
    -> HeaderHash
    -> [HeaderHash]
    -> m ()
getProcessBlocks pm txpConfig diffusion nodeId desired checkpoints = do
```
which suggests that the second last parameter is *different* from the last parameter.

Comment (yihuang) [reply lib/src/Pos/Network/Block/Retrieval.hs]: Look at line 113, the hash is already calculated, it's intent to download a single block, so these parameters are the same.

**Comment by jcmincke** [comment]
@erikd Do we merge it or is the fix already in develop?

**Comment by erikd** [comment]
Closing this in favor of #3565 so it can be run thought CI.

**Pull request closed without merging**